ArcGIS.md

```diff
@@ -71,6 +74,19 @@ Most of the data for this tutorial originated from federal government websites.
 IMPORTANT TIP: Make sure there are no empty spaces within the columns and rows of your spreadsheets. ArcGIS will only read csv or tsv files that are computer readable, NOT human readable. If you have any blank cells within your spreadsheet 
 
 ![Example of  the Excel spreadsheet](ArcGIS-Imgs/spreadsheet.PNG)
-## Adding Data to your Map: 
-## Display Data and Tables on your Map: 
+## Adding and Displaying Data on your Map:
+1. Go to the Map Tab on the Top left and Add Data section. When the drop down menu displays, look for the XY Point Data option.
+1. You will see a panel pop up on the right side of the screen in the Geoprocessing section. Select your Input Table within your local file directory or through Living Atlas.
+1. The blank sections should autofill but still make sure the X Field contains the Longitude coordinates and the Y Field contains the Latitude coordinates.
+1. You can use the auto generated Coordinate System, but if needed for your project select the best Coordinate System for you. 
+1. Click the run button in the button right corner of the Geoprocessing section.
+You will see the data display on the map with basic points.
+
+##How to Export or Upload your Project:
+This tutorial provides detailed steps to [create a project package](https://pro.arcgis.com/en/pro-app/2.7/help/sharing/overview/project-package.htm) as well as upload the project package to your online account.
+
```

Comment:
Suggestion for a new short section: Troubleshooting!
* Common problems and Solutions (If Available)
     * ARC GIS updates and suddenly it looks like your data isn't properly formatted: (Solution: reupload your data)
     * Anything else? (Finicky Timeline: Choose a year, and ArcGIS displays the last day of the previous year)
     *

Reply:
Comment somewhere on needing to Expect Annoying Troubles. 
How to Google-Search solutions to your problems.